subprojects/gradle/cd/src/test/kotlin/com/avito/ci/RegisterUiTestConfigurations.kt

```diff
@@ -0,0 +1,55 @@
+package com.avito.ci
+
+internal fun registerUiTestConfigurations(vararg names: String, isPerformance: Boolean = false): String {
+    val configurations = names.map { name ->
+        """$name {
+            
+            ${if (isPerformance) "performanceType = com.avito.instrumentation.configuration.InstrumentationConfiguration.PerformanceType.SIMPLE" else ""}
```

Comment:
Is `performance` deleted?

Reply:
not for now (backward compatibility); will delete it in next version along with this line